src/result.rs

```diff
@@ -70,71 +75,93 @@ pub enum CqlType {
 #[napi]
 impl QueryResultWrapper {
     pub fn from_query(internal: QueryResult) -> QueryResultWrapper {
-        QueryResultWrapper { internal }
+        let value = match internal.clone().into_rows_result() {
+            Ok(v) => QueryResultVariant::RowsResult(v),
+            Err(_) => QueryResultVariant::EmptyResult(internal),
+        };
```

Comment:
:wrench: IIRC, `into_rows_result` may fail in two cases:
1. if the underlying Result response is not of Rows kind (correctly handled here)
2. if deserialization of Result metadata fails. In such case, the error should be propagated, not converted into `QueryResultVariant::EmptyResult`, because the current handling is basically silent ignoring of errors.